gnocchi/tests/functional/gabbits/archive.yaml

```diff
@@ -49,33 +55,21 @@ tests:
       POST: /v1/archive_policy
       request_headers:
           content-type: text/plain
-          x-roles: admin
+          # User admin
+          authorization: "basic YWRtaW46"
       status: 415
       response_strings:
           - Unsupported Media Type
 
     - name: wrong method
       PUT: /v1/archive_policy
       request_headers:
-          content-type: application/json
-          x-roles: admin
+          # User admin
+          authorization: "basic YWRtaW46"
       status: 405
 
-    - name: wrong authZ
-      POST: /v1/archive_policy
-      request_headers:
-          content-type: application/json
-          x-roles: clancy
```

Comment:
I've changed this to pass an empty authorization, since I set it by default to be present I cannot remove it. Gabbi does not allow that AFAICS. Not a big deal I guess.